testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/page/v3/PageIT.java

```diff
@@ -52,4 +52,10 @@ public void testResponsiveGridStructure() throws ClientException {
         String content = adminAuthor.doGet("/content/core-components/simple-page/simple-page-v3.html", 200).getContent();
         GraniteAssert.assertRegExFind(content, "Basic Title");
     }
+
+    @Test
+    public void testHeadJSLibs() throws ClientException {
+        String content = adminAuthor.doGet("/content/core-components/simple-page/simple-page-v3.html", 200).getContent();
+        GraniteAssert.assertRegExFind(content, "<script src=\"/etc.clientlibs/foundation/clientlibs/jquery.js\"></script>");
```

Comment:
This pattern will most probably not work with the long-term caching suffixes (see https://github.com/adobe/aem-core-wcm-components/blob/main/testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/ClientlibsIncludeIT.java#L44 for a possible solution)